eng/common/TestResources/New-TestResources.ps1

```diff
@@ -161,7 +161,12 @@ try {
 
     # Enumerate test resources to deploy. Fail if none found.
     $repositoryRoot = "$PSScriptRoot/../../.." | Resolve-Path
-    $root = [System.IO.Path]::Combine($repositoryRoot, "sdk", $ServiceDirectory) | Resolve-Path
+    
+    $root = $repositoryRoot | Resolve-Path
```

Comment:
Might as well combine this with repositoryRoot variable above which is already resolved

Reply:
aah... yup